src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/AuthDAO.scala

```diff
@@ -22,6 +22,12 @@ trait AuthDAO {
     (loadUserByEmail(email, txn).map(Left(_)) ++ loadGroupByEmail(email, txn).map(Right(_))).headOption
   }
 
```

Comment:
as noted in other comments, add and remove are higher order functions and should be at the service level